tests/mapper/test_node_mappers.py

```diff
@@ -904,3 +904,30 @@ def test_generator_methods(self):
             ppnp_sparse_failed = True
 
         assert ppnp_sparse_failed
+
+
+@pytest.mark.parametrize("sparse", [True, False])
+def test_corrupt_full_batch_generator(sparse):
+
+    G, _ = create_graph_features()
+
+    generator = FullBatchNodeGenerator(G, sparse=sparse)
+
+    base_gen = generator.flow(G.nodes())
+    gen = CorruptedNodeSequence(base_gen)
+
+    [shuffled_feats, features, *_], targets = gen.__getitem__(0)
```

Comment:
```suggestion
    [shuffled_feats, features, *_], targets = gen[0]
```

Reply:
Did you forget?

Reply:
ah.. yeah basically. Added this in, broke tests (not related to this change), removed this, and and then forgot to add it back in!

Reply:
Still not added?!

Reply:
what?? I have the change locally, I'l double check with github this time